app/controllers/peer_reviews_controller.rb

```diff
@@ -89,6 +90,13 @@ def unassign(reviewers_to_remove_from_reviewees_map)
         pr.destroy
       end
     end
+
+    # Mass unassigning comes last, reason being that the above code does not
+    # have to worry about find failing and returning nil.
+    selected_reviewee_group_ids.each do |reviewee_id|
+      result = Grouping.find(reviewee_id).current_submission_used.get_latest_result()
```

Comment:
This line (which is similar to one above in the old code) should be able to be simplified using an association off of groupings for peer reviews.